CONTRIBUTING.md

```diff
@@ -53,8 +54,8 @@ Primarily DAB uses integration tests to verify the engine operates correctly in
 To run the SQL tests locally you will need to:
 
 1. Setup a database using the server(s) that you want to test against.
-1. Create the database schema using the `DatabaseSchema-<engine>.sql` file.
-1. Set the connection string in `dab-config.<engine>.json` or use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/azure/data-api-builder/configuration-file#accessing-environment-variables).
+1. Create the database schema using the `src/Service.Tests/DatabaseSchema-<engine>.sql` file.
+1. Set the connection string in `src/Service.Tests/dab-config.<engine>.json` or use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/azure/data-api-builder/configuration-file#accessing-environment-variables).
```

Comment:
Should this all be listed as `1.` ?

Reply:
Just checked this ! It is rendering 1,2,3 correctly!

Reply:
Neat markdown feature :)